common/types/block.go

```diff
@@ -86,7 +86,7 @@ type MiniBlock struct {
 }
 
 func (id BlockID) AsHash32() Hash32 {
```

Comment:
are we going to change tx ids and atx ids as well ? if so we really don't need to keep AsHash32 only for sync proposes

Reply:
No, I don't think we should change everything to be 20bytes hash

Reply:
we should decrease them as well (maybe not in this pr), maybe even smaller than 20, as iddo wants

Reply:
We should change them also. Another PR may address the on-wire vs real-size.
For now, we want to have significantly smaller IDs across the system.

Reply:
IMO, we should change tx and atx IDs to Hash20 now.
Later, on another PR, we should distinguish between wire-size and internal-size.

Reply:
why? it's already implemented this way

Reply:
Right, I thought we changed it too but it was already like that.